nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# Create new products drop down menu has product types and an error message.

Fixes:
-No drop down menu with product types
-No error message if drop down is left blank

Changes proposed in this request & reasons behind organizational or architectural decisions:
-Drop down menu now displays all of the product types
-If the drop down is left empty an error message is displayed 

Steps to test:
Create a product
-Open the program
-Select the create new button / sell something in the nav bar
-Fill out the currently displayed form
-Submit to create a new product for sale 

Error message for product
-Open the program
-Select the create new button / sell something in the nav bar
-Fill out the currently displayed form except for product categories 
-submit

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-
-

Commit message:
-

Link to feature ticket:
-

@mighty-mahi-mahi
